README.md

```diff
@@ -5,21 +5,16 @@
 If you work on more than one feature at a time, you are guaranteed to multiply your bugs and your anxiety.
 
 ## Making a plan
```

Comment:
Great Job! You made a plan and followed through with each step.. This app is amazing!